docs/src/maintain/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How ESLint is Maintained
+eleventyNavigation:
+    key: how eslint is maintained
+    parent: maintain eslint
+    title: How ESLint is Maintained
+    order: 1
+
+---
+
+This page explains the different roles and structures involved in maintaining ESLint.
+
+## The ESLint Team
+
+The ESLint team work together to develop and maintain ESLint. The team consists of several different roles, including:
+
+* **Technical Steering Committee (TSC)**: Manage releases, review feature requests, and meet regularly to ensure ESLint is properly maintained.
+* **Reviewers**: Review and implement new features.
+* **Committers**: Review and fix bugs and help triage issues.
+* **Website Team**: Focus specifically on [eslint.org](https://eslint.org).
+* **Support Team**: Help in Discord and discussions.
+
+To learn more about the current team members, refer to the [Team](https://eslint.org/team/) page.
+
+## Organization Structure
+
+TODO: provide a little more info about what the OpenJS foundation actually does vis-a-vis ESLint.
```

Comment:
I'm not sure if we'd want to use this verbatim, but here's an overview. You can pick and choose from this:

The OpenJS Foundation provides the legal infrastructure for JavaScript projects like ESLint. It is the owner of the intellectual property related to ESLint, including copyrights and trademarks, and ensures the independence of the project.

While the OpenJS Foundation does not participate in the day-to-day functioning of the project, they are a resource for ESLint should we need legal advice or representation.

Reply:
thx, this is perfect. adapting into the answer.